routes/cache.js

```diff
@@ -71,4 +102,24 @@ router.put('/item', function (req, res) {
   })
 })
 
+// PUT multiple cache items
+router.put('/items', function (req, res) {
+  const items = JSON.parse(req.body)
+  const memcached = req.app.locals.memcached
+  if(!Array.isArray(items)) {
+    return res.status(400).json(jerr.BadRequest)
+  }
+
+  return Promise.allSettled(items.map(i => setPromise(memcached, i)))
+  .then(results => {
+    if(results.find(r => r.status === 'rejected')) {
+      res.status(500).json(jerr.InternalServer)
```

Comment:
This line could `throw` an error. Then the `else` is not needed and the `catch` below would send the response to the client.

Reply:
Done!